odxtools/isotp_state_machine.py

```diff
@@ -128,7 +131,7 @@ async def read_telegrams(self, bus: Union[can.Bus,
                 for tmp in self.decode_rx_frame(msg.arbitration_id, msg.data):
                     yield tmp
         else:
-            assert isinstance(bus, TextIO)
+            assert isinstance(bus, (TextIOBase, TextIOWrapper))
```

Comment:
I suppose that this can become
```suggestion
            assert isinstance(bus, TextIOBase)
```

because `TextIOWrapper` derives from `TextIOBase`...

Reply:
(note that, to satisfy the CI, you also need to remove the now unused import of `TextIOWrapper` above...)

Reply:
I agree. `TextIOWrapper` is indeed derived from `TextIOBase` (src: https://docs.python.org/3/library/io.html#io.TextIOWrapper)